start.sh

```diff
@@ -172,7 +172,7 @@ else
 
     if [ -z "$NoViaVersion" ]; then
         # Update ViaVersion if new version is available
-        ViaVersionVersion=$(curl --no-progress-meter -k -L -H "Accept-Encoding: identity" -H "Accept-Language: en" -L -A "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4.212 Safari/537.36" https://ci.viaversion.com/job/ViaVersion/lastBuild/artifact/build/libs/ | grep 'href="ViaVersion' | cut -d'"' -f2)
```

Comment:
It seems https://ci.viaversion.com/job/ViaVersion/lastBuild/artifact/build/libs/ become minified and thus extracting of 2nd field fails